lib/bundler/cli/add.rb

```diff
@@ -2,34 +2,38 @@
 
 module Bundler
   class CLI::Add
+    attr_reader :gems, :options, :version
```

Comment:
I created the attributes reader, this allow to avoid to call every time the variables with @

Reply:
I added version, multiples usages.